operator/src/main/resources/scripts/utils.py

```diff
@@ -61,3 +63,45 @@ def trace(arg1,arg2='SENTINEL'):
     traceInner('FINE',arg1)
   else:
     traceInner(arg1,arg2)
+
+def get_server_template_listening_ports_from_configxml(config_xml):
+  '''
```

Comment:
This function seems pretty specialized. Does it need to be in 'utils'?

Reply:
I will move this to introspectDomain.py